docusaurus.config.js

```diff
@@ -131,6 +131,11 @@ const config = {
         disableSwitch: true,
         respectPrefersColorScheme: true,
       },
+      metadata: [
+        { property: 'og:description', content: 'Developer Docs for passwordless authentication' },
+        { property: 'og: image', content: '/assets/zero-trust-passwords.webp' },
+        { name: 'twitter:card', content: 'Beyond Identity Developer Docs' }
```

Comment:
nit: What's with the extra space here? `'og:description'`,  `'twitter:card'` vs `'og: image'` 😛

Reply:
Lol nice catch. Fixed